src/App.jsx

```diff
@@ -42,14 +42,15 @@ class App extends Component {
             <Route path="/" exact component={HomePage} />
             <Route path="/login" component={LoginPage} />
             <Route path="/article" component={ArticlePage} />
+            <Route path="/verify" component={VerifyUser} />
             <Route path="/signup" component={SignupPage} />
           </Switch>
-          <ProfileNavBar
+          {/* <ProfileNavBar
```

Comment:
Kindly remove the commented block. thank you

Reply:
Thanks, I've done that.